search/result_processor.py

```diff
@@ -0,0 +1,146 @@
+""" overridable result processor object to allow additional properties to be exposed """
+import inspect
+from itertools import chain
+import json
+import logging
+import re
+import textwrap
+
+from django.conf import settings
+from django.core.serializers.json import DjangoJSONEncoder
+
+from .utils import _load_class
+
+DESIRED_EXCERPT_LENGTH = 100
+ELLIPSIS = "&hellip;"
```

Comment:
I seem to recall hellip are discouraged from the accessibility folks. Maybe a cleanup task?